server/routes/redirects.py

```diff
@@ -66,6 +66,11 @@ def get_involved():
     return redirect('https://docs.datacommons.org/contributing/', code=302)
 
 
+@bp.route('/tools/stat-var')
```

Comment:
If this tool has not been exposed yet, this is not needed? Also this only works for prod but not other deploy.

Reply:
I'll keep for now since /tools/stat-var currently works on prod
note: this will temporarily break autopush.../tools/stat-var until this change is pushed to prod